pkg/plugin/hnsstats/vfp_counters_windows.go

```diff
@@ -152,15 +152,21 @@ func getVfpPortCountersRaw(portGUID string) (string, error) {
 
 	cmd := exec.Command("cmd", "/c", vfpCmd)
 	out, err := cmd.Output()
+	if err != nil {
+		return "", errors.Wrap(err, "errored while running vfpctrl /get-port-counter")
```

Comment:
an error occurred, produced an error, ..... errored comes off as technical slang. I treat this like "til"